tools/Autotag/README.md

```diff
@@ -0,0 +1,5 @@
+# Autotag
+
+Use of this tagging scripts requires the project to have a changelog as described in the confluence link below
+
+https://confluence.amd.com/display/MLSE/Changelogs+and+Release+Notes
```

Comment:
That's an internal link. Let's make the changelog requirements public. I can handle that separately.

Reply:
Let's not merge until the changelog guidelines are public.